### DISCUSSION TITLE
IWP 결제오류

### DISCUSSION THREAD

**Discussion by Andwhy81kr** [naverz/zepeto-studio-kor#807] [4. General]
1.3.0 SDK, App버전 3.12.0

기존의 이슈 그대로 있습니다.
https://github.com/naverz/zepeto-studio-kor/discussions/558
1.3.0에서 수정되었다고 리포트 받았지만 여전히 남아있네요.

확인부탁드립니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, IWP 부분 확인해 보겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
결제 관련해서 불편을 드려 죄송합니다.
내부에서 확인 결과 관련 디펜던시 문제들이 발견되어서 개선 예정입니다.
일정을 확답 드릴 수는 없으나 최우선 순위 작업으로 진행 중에 있습니다. 
감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 금일 안드로이드 앱 버전 3.13.6 핫픽스 배포로 IWP 이슈 수정되었습니다.
감사합니다. 🙏
